GCS_README.rst

```diff
@@ -0,0 +1,31 @@
+PyPI Cloud - GCS Backend
+============
+Installation
+============
+
+Create a new virtualenv for the project and activate it
+
+Download the package, install with python setup.py install - this will also handle basic requirements.
+
+Working with GCS required a fix of gcs-oauth2-boto-plugin, install the gcs-oauth2-boto-plugin-hexadite version.
```

Comment:
I searched pypi for `gcs-oauth2-boto-plugin-hexadite` and didn't find anything. Where does this exist or how can the user obtain it?